common/djangoapps/student/tests/test_password_policy.py

```diff
@@ -160,6 +160,44 @@ def test_enough_wordss(self):
         obj = json.loads(response.content)
         self.assertTrue(obj['success'])
 
+    @patch.dict("django.conf.settings.PASSWORD_COMPLEXITY", {'NUMERIC': 3})
+    def test_not_enough_numeric_characters(self):
+        self.url_params['password'] = u'thishouldfail½2'
+        response = self.client.post(self.url, self.url_params)
+        self.assertEqual(response.status_code, 400)
+        obj = json.loads(response.content)
+        self.assertEqual(
+            obj['value'],
+            "Password: Must be more complex (must contain 3 or more numeric characters)",
+        )
+
+    @patch.dict("django.conf.settings.PASSWORD_COMPLEXITY", {'NUMERIC': 3})
+    def test_enough_numeric_characters(self):
+        self.url_params['password'] = u'thisShouldPass½34'  # This unicode 1/2 should count as a numeric value here
```

Comment:
Could you make the 34 be a 33 or something, to confirm that we don't care about uniqueness for NUMERIC?

(Same for ALPHABETIC tests too.)